R/0_documentation.R

```diff
@@ -19,13 +19,13 @@ rd_family <- function(usage = NULL) {
   paste("A character string specifying the family for modeling",
         ifelse(usage == "combined", paste(" seasonal onset.")))
 }
+rd_only_current_season <- "A logical determining if the output should only include results for the current season (TRUE)
+or if the output should include results for all available seasons (FALSE)"
```

Comment:
I have seen this more concise style being used. 
Consider it as an alternative:

```suggestion
rd_only_current_season <- "Should the output only include results for the current season?"
```